workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/dashboard/MDFRequestChart/index.tsx

```diff
@@ -69,7 +69,10 @@ const MDFRequestChart = () => {
 				)
 			));
 
-		const currency = account ? account.currency : 'USD';
+		const currency =
+			account && !myUserAccount.organizationBriefs.length
```

Comment:
should we just be checking if account?.currency exists? or what is the benefit of using !myUserAccount.organizationBriefs.length ?

Reply:
> should we just be checking if account?.currency exists?
Probably this would be enough, but when i was testing, i noticed that since my user is bonded to a account (Deathray) it uses the account's currency. I don't know if all channel member are not signed to a account (maybe that are some for test purpose), so i add this verification in order to catch this misbehavior.

> what is the benefit of using !myUserAccount.organizationBriefs.length ?
This is the same verification used in the useIsChannel hook, so i just put this inline instead of import another hook to the component. Using the hook this part would be !isChannel.

Reply:
1. I see, okay that is probably fine then
2. hmm, it might be good to use the isChannel hook, for clarity and in case we update the logic there at any point. I am not completely sure why that logic even works, but I would have to look into it

Reply:
I agree with you, i think we should be using roles for this purpose.

Reply:
Done!